pages/RapPage.js

```diff
@@ -30,31 +30,49 @@ export default function RapPage() {
   return (
     <>
       <Header />
-      <StyledDiv>
+      <StyledHeadDiv>
         <BackButton />
         <ShoppingCart />
-      </StyledDiv>
-      <div>
-        <h2>RAP</h2>
-      </div>
+      </StyledHeadDiv>
+      <StyledRapLineDiv>
+        <StyledRapP>RAP</StyledRapP>
+        <StyledHr />
+      </StyledRapLineDiv>
       <div>
         <ul>
           {data.map((rapstars) => (
-            <li key={rapstars._id}>
-              <Image
+            <StyledLi key={rapstars._id}>
+              <StyledImage
                 src={rapstars.image}
-                width={"280"}
-                height={"150"}
+                width={"550"}
+                height={"300"}
                 alt="Rap-Artist"
+                priority="high"
+                style={{
+                  position: "relative",
+                  zIndex: -1,
+                }}
               />
-              <p>{rapstars.rating}</p>
-              <h4>{rapstars.name}</h4>
-
-              <p>
+              <StyledRatingP>{rapstars.rating}</StyledRatingP>
+              <StyledArtistnameP>{rapstars.name}</StyledArtistnameP>
+              <StyledPriceP>
                 {rapstars.currency} {rapstars.price}
-              </p>
-              <Link href={`/rapDetails/${rapstars._id}`}>Find Tickets</Link>
-            </li>
+              </StyledPriceP>
+              <Link
+                href={`/rapDetails/${rapstars._id}`}
+                style={{
+                  textDecoration: "none",
+                  color: "#139df4",
+                  position: "relative",
+                  bottom: "10px",
+                  left: "200px",
+                  fontSize: "15px",
+                  fontWeight: "bold",
+                }}
+              >
```

Comment:
Avoid inline styling here as well. Use `styled-components` for styling the `Link`.